specification/azsadmin/resource-manager/containerregistry/Microsoft.ContainerRegistry.Admin/preview/2019-11-01/quotas.json

```diff
@@ -0,0 +1,271 @@
+            }
+          },
+          "default": {
+            "description": "Error Response.",
+            "schema": {
+              "$ref": "containerregistry.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerRegistry.Admin/locations/{location}/quotas": {
+      "get": {
+        "x-ms-examples": {
+          "Returns a list of container registry quotas at the given location.": {
+            "$ref": "./examples/Quotas/List.json"
+          }
+        },
+        "tags": [
+          "ContainerRegistryQuotas"
+        ],
+        "description": "Returns a list of container registry quotas at the given location.",
+        "operationId": "ContainerRegistryQuotas_List",
+        "parameters": [
+          {
+            "$ref": "containerregistry.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "containerregistry.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "containerregistry.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- The list of container registry quotas has been returned.",
+            "schema": {
+              "$ref": "#/definitions/ContainerRegistryQuotaList"
+            }
+          },
+          "default": {
+            "description": "Error Response.",
+            "schema": {
+              "$ref": "containerregistry.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ContainerRegistryQuota": {
+      "description": "Container registry quota.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "description": "Container registry quota properties.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ContainerRegistryQuotaProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "containerregistry.json#/definitions/Resource"
+        }
+      ]
+    },
+    "ContainerRegistryQuotaProperties": {
+      "description": "Container registry quota properties.",
+      "type": "object",
+      "properties": {
+        "numberOfRegistries": {
+          "description": "Total number of container registry accounts.",
+          "type": "integer",
+          "format": "int32",
+          "default": 20
+        },
+        "capacityPerRegistryInGb": {
+          "description": "Storage capacity (GB) of each registry.",
+          "type": "integer",
+          "format": "int32",
+          "default": 50
+        }
+      }
+    },
+    "ContainerRegistryQuotaList": {
+      "description": "List of container registry quotas.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of container registry quotas.",
+          "type": "array",
```

Comment:
paging at some point?

Reply:
this set doesn't have the pagination actually

Reply:
We wont have paging for the quota api any time. its going to be single digit most likely